### PR TITLE
Quay

### PR DESCRIPTION
**Pull request by EItanya**

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -90,8 +90,8 @@ steps:
   # 3) Publish helm chart
   - name: 'gcr.io/cloud-builders/docker'
     entrypoint: 'bash'
-    args: ['-c', 'docker login --username soloiobot --password $$DOCKER_HUB_PASSWORD']
-    secretEnv: ['DOCKER_HUB_PASSWORD']
+    args: ['-c', 'docker login quay.io --username "solo-io+solobot" --password QUAY_IO_PASSWORD']
```
Comment (rickducott) [inline cloudbuild.yaml]: do we need $$?

Comment (EItanya) [reply cloudbuild.yaml]: yup, double click accidentally deleted it

File: install/helm/supergloo/generate-values.go [hunk 2]
```diff
@@ -17,11 +20,17 @@ const (
 	chartTemplate = "install/helm/supergloo/Chart-template.yaml"
 	chartOutput   = "install/helm/supergloo/Chart.yaml"
 
+	glooPkg = "github.com/solo-io/gloo"
```
Comment (rickducott) [inline install/helm/supergloo/generate-values.go]: Use `versionutils.GlooPkg` instead

**Pull request merged**